ui/src/apis/beaconchaApi/buildValidatorSummaryURL.ts

```diff
@@ -8,14 +8,22 @@ export default function buildValidatorSummaryURL({
   allValidatorsInfo: BeaconchaGetResponse[];
   network: string;
 }): string {
-  let summaryValidatorURL =
-    beaconchaApiParamsMap.get(network)!.baseUrl + "/dashboard?validators="; //TODO Be careful with the !
+  if (!beaconchaApiParamsMap.has(network)) {
+    throw new Error(`Invalid network: ${network}`);
+  }
 
-  allValidatorsInfo.forEach((validatorInfo) => {
-    summaryValidatorURL += validatorInfo.data.validatorindex + ",";
-  });
+  const baseUrl = beaconchaApiParamsMap.get(network)?.baseUrl;
+
+  if (baseUrl == null) return "";
```

Comment:
This check is already done above, you must use a var to store it and do the check only once:
```
const baseUrl = beaconchaApiParamsMap.get(network)?.baseUrl;
if(!baseUrl)   throw new Error(`Invalid network: ${network}`);
  
```